vms/proposervm/proposer/windower.go

```diff
@@ -51,6 +62,23 @@ type Windower interface {
 		validatorID ids.NodeID,
 		maxWindows int,
 	) (time.Duration, error)
+
+	ExpectedProposer(
+		ctx context.Context,
+		chainHeight,
+		pChainHeight uint64,
+		parentBlockTime,
+		blockTime time.Time,
+	) (ids.NodeID, error)
+
+	MinDelayForProposer(
```

Comment:
Can we document these calls (Especially `MinDelayForProposer`)?

Reply:
Done. LMK if explanation clarifies usage